guide/getting-started/getting-started.md

```diff
@@ -34,70 +34,68 @@ Illustration sources
+Bitcoin is open-source, decentralized, permissionless, fungible, stateless, and non-custodial. If those words don’t mean anything to you, consider starting here.  
 
 ---
 
 ### [Open design]({{ '/guide/getting-started/open-design/' | relative_url }})
 
-Open design is about global collaboration that balances the free exchange of design information, the independence of individual designers and the collective power of collaboration.
+Bitcoin isn’t just changing money—it’s changing how designers collaborate across broad use cases. While open-source development has been around for years, open design is brand new.
 
 ---
 
 ### [The visual language of Bitcoin]({{ '/guide/getting-started/visual-language/' | relative_url }})
 
-As an open and public protocol, Bitcoin does not have a strictly defined brand. Yet there is a recognizable symbol that has evolved and remixed endlessly.
+Since bitcoin is public, open, and dependent upon no single person or organization, its brand is still undefined—and it might always be. But there’s still a to learn about how best to represent it visually.
 
 ---
 
 ### [User research]({{ '/guide/getting-started/user-research/' | relative_url }})
 
-All products begin with user needs and end in those same users hands. User research is a fundamental activity that helps us focus on things people care about.
+Before a product can solve its users’ problems, its designers need to understand what those problems are. User research helps us zoom in on what users care about.
 
 ---
 
 ### [Personal finance]({{ '/guide/getting-started/personal-finance/' | relative_url }})
 
-If we consider Bitcoin money, we need to consider how Bitcoin users think about their finances. This page provides a simple framework that can serve as a starting point, deriving bitcoin software and hardware setups from common budgeting categories.
+The language of bitcoin is the language of money. If you’re still new to the concepts that underpin personal finance, this section can get you up to speed.
 
 ---
 
 ### [Usage life cycle]({{ '/guide/getting-started/usage-life-cycle/' | relative_url }})
 
-All users start without any knowledge of Bitcoin or your particular product. As we learn more, our understanding and needs change. As designers, it is essential to guide users through this experience.
+As familiarization with the bitcoin ecosystem increases, users’ needs evolve. As designers, we need to understand each of the steps along that journey.
 
 ---
 
 ### [Common user flows]({{ '/guide/getting-started/common-user-flows/' | relative_url }})
 
-Most Bitcoin applications share certain user activities, from initial setup to making a transaction. It is vital to follow users' expected patterns and behaviors to reduce mistakes and enable seamless switching between products.
+Because solutions to common problems such as how to send, store, and purchase bitcoin are often the most in demand, they are also the most available as a resource online.
```

Comment:
`Most in demand` by whom? Also not sure what is meant by `they are also the most available as a resource online`?

Goal of this page is to provide a designer an overview of everything that can happen in a Bitcoin application. Some of those activities are totally not obvious and may take a lot of time to identify on your own. So it's a bit of a cheat sheet that should allow designers to make better decisions because they have a good idea of all user flows that need to be considered. Especially useful when considering navigation hierarchy.

Reply:
Reverted https://github.com/BitcoinDesign/Guide/pull/221/commits/6a0b33bed7ccef80c67962885e4b9f3a99796e09